generic/csp/write-in-shm-dir.yaml

```diff
@@ -2,21 +2,26 @@ apiVersion: security.kubearmor.com/v1
 kind: KubeArmorClusterPolicy
 metadata:
   name: write-in-shm-dir
+  annotations:
+    app.accuknox.com/type: harden
 spec:
-  action: Block
+  action: Audit
   file:
     matchDirectories:
-    - dir: /dev/shm/
-      readOnly: true
-      recursive: true
-  message: Alert! write to /dev/shm folder prevented.
+      - dir: /dev/shm/
+        readOnly: true
+        recursive: true
+  message: "Detected attempt to write to /dev/shm folder"
   selector:
     matchExpressions:
-    - key: namespace
-      operator: NotIn
-      values:
-      - kube-system
-  severity: 5
+      - key: namespace
+        operator: NotIn
+        values:
+          - kube-system
+  severity: 7
   tags:
-  - MITRE
-  - MITRE_execution
+    - MITRE
+    - MITRE_T1059_command_and_scripting_interpreter
+    - MITRE_T1543_create_or_modify_system_process
+    - shm-protection
+    - harden
```

Comment:
```suggestion
```

Reply:
I suggest removing this as we already know the policy audits shm. 
Also the tag says as shm-protection. But it is an Audit.